tide-whisperer.go

```diff
@@ -203,11 +205,57 @@ func main() {
 		res.Write([]byte("OK\n"))
 		return
 	}))
+	
+	/*
```

Comment:
not that we are using godocs at the moment but in golang comments are normally done as out lined below so you can generate dev docs from them 

```
// my comment
// .... and yet more comments 
// ... etc
```